lib/controller.js

```diff
@@ -599,6 +599,27 @@ controller.updateTesselWithVersion = function(force, tessel, currentVersion, bui
     });
 };
 
+controller.tesselFirmwareVerion = function(opts) {
+  return Tessel.get(opts)
+    .then(function(tessel) {
+      return tessel.fetchCurrentBuildInfo()
+        .then(function(versionSha) {
+          updates.requestBuildList().then(function(builds) {
```

Comment:
You'll need to `return` this value otherwise the process will close before this Promise is resolved (at least it does for me):

```.js
return updates.requestBuildList().then(function(builds) {
...
});
```